react-web/src/components/_homepage.scss

```diff
@@ -0,0 +1,17 @@
+.homepage {
+  padding-top: 20px;
```

Comment:
Can we use https://getbootstrap.com/docs/4.1/utilities/spacing/ or https://github.com/patternfly/patternfly-css/blob/master/CODE-GUIDELINES.md#spaces-margins-and-padding?

Reply:
The spacing for bootstrap won't be available since we're using v3.3.x. The patternfly spacing is also not available to the patternfly library we are using.

Reply:
:+1: